linkis-web/src/apps/linkis/assets/styles/console.scss

```diff
@@ -17,8 +17,29 @@
  
  @charset "UTF-8";
  @import '@/common/style/variables.scss';
+ @import './hint.min.css';
+//  * {
 
+//     font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "JinbiaoSong", "JinbiaoSongExt";
+//   }
+body {
+    font-family: "Helvetica Neue",Helvetica,"PingFang SC","Hiragino Sans GB","Microsoft YaHei","微软雅黑",Arial,sans-serif, "JinbiaoSong", "JinbiaoSongExt";
+}
 .console-page{
+    .ivu-input {
+        font-family: Arial, -apple-system, BlinkMacSystemFont, "Segoe UI", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "JinbiaoSong", "JinbiaoSongExt";
```

Comment:
why use another font 'Arial'

Reply:
It's style of iview input component, I just supplement jinbiaosong and jinbiaosongext font